extensions/amp-form/0.1/amp-form.js

```diff
@@ -355,6 +356,13 @@ export class AmpForm {
     });
   }
 
+  /** @private */
+  installInputMasking_() {
+    Services.inputmaskServiceForDoc(this.form_).then(inputmaskService => {
```

Comment:
should this be `inputmaskServiceForDocOrNull` since extension is optional? I don't know what currently happens if inputmask extension is not included. My guess is promise is rejected and error gets logged as uncaught promise rejection?